src/app/testing/gene-mocks.ts

```diff
@@ -252,26 +252,26 @@ export const mockLinksListResponse: LinksListResponse = {
   items: [mockDataLink1, mockDataLink2],
 };
 
-export const mockExpValidation: Array<GeneExpValidation> = [
+export const mockExpValidation: GeneExpValidation[] = [
   {
     _id: '12345',
     ensembl_gene_id: 'string',
     hgnc_symbol: 'BBB',
-    Hypothesis_tested: 'string',
-    Summary_findings: 'string',
-    Published: 'Yes',
-    Species: 'string',
-    Model_system: 'string',
-    Outcome_measure: 'string',
-    Outcome_measure_details: 'string',
-    Contributors: 'John D',
-    Team: 'Duke',
-    Reference_doi: 'https://doi.org/10.15252/abc.123',
-    Date_report: '01/01/2021',
+    hypothesis_tested: 'string',
+    summary_findings: 'string',
+    published: 'Yes',
+    species: 'string',
+    model_system: 'string',
+    outcome_measure: 'string',
+    outcome_measure_details: 'string',
+    contributors: 'John D',
+    team: 'Duke',
+    reference_doi: 'https://doi.org/10.15252/abc.123',
+    date_report: '01/01/2021',
   },
 ];
 
-export const mockDistributionData = {
+export const mockDistributionData: DistributionData = {
```

Comment:
The DistributionData import wasn't being used, and all the other mocks do this...